### PR TITLE
docs/troubleshooting.md: Update for 0.6.0 release

### PR DESCRIPTION
**Pull request by alexbrand**
Updates #644 

- Adds note about visualizing the DAG via the DOT debug endpoint
- Adds note about cert-manager being preferred over kube-lego

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -57,7 +72,9 @@ Replace `contour cli lds` with `contour cli rds` for RDS, `contour cli cds` for
 
 ## Can't make kube-lego work with Contour
 
-If you use [kube-lego][0] for Let's Encrypt SSL certificates, kube-lego appears to set the ingress class on the ingress record it uses for the acme-01 challenge to `nginx`.
+[Cert-manager][8] is preffered over [kube-lego][0] for using Let's Encrypt SSL certificates with Contour.
+
+If you must use kube-lego for Let's Encrypt SSL certificates, kube-lego appears to set the ingress class on the ingress record it uses for the acme-01 challenge to `nginx`.
```
Comment (stevesloka) [inline docs/troubleshooting.md]: I think we should just remove all kube-lego references and show how to use cert-manager.

Comment (alexbrand) [reply docs/troubleshooting.md]: That works for me! How to use cert-manager belongs outside of the troubleshooting docs, so I won't include that in this PR.

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -39,6 +39,21 @@ CONTOUR_POD=$(kubectl -n heptio-contour get pod -l app=contour -o jsonpath='{.it
...
+
+To download the graph and save it as a PNG:
+
+```sh
+# Port forward into the contour pod
+CONTOUR_POD=$(kubectl -n heptio-contour get pod -l app=contour -o jsonpath='{.items[0].metadata.name}')
+# Do the port forward to that pod
+kubectl -n heptio-contour port-forward $CONTOUR_POD 6060
+# Download and store the DAG in png format
+curl localhost:6060/debug/dag | dot -T png > contour-dag.png
```
[the remark this answers is not in the document]
Comment (alexbrand) [reply docs/troubleshooting.md]: Good call! Thanks. I'll add an example.

**Comment by alexbrand** [comment]
@stevesloka Updated. PTAL

**Pull request merged**